src/components/admin/ImageController.jsx

```diff
@@ -438,21 +446,20 @@ export default class ImageUploader extends BaseComponent {
         >
           Please enter a new name for the image:
           <br />
-          <TextField
-            name="newName"
-            defaultValue={this.state.nameInput}
-            onChange={(event) => {
-              this.safeSetState({ nameInput: event.target.value });
-            }}
-          />
+          <form onSubmit={this.changeImageName}>
+            <TextField
+              name="newName"
+              fullWidth={this.state.fullWidthTextField}
```

Comment:
We don't put constants in state, only mutables, so you actually just have to write `fullWidth` (which is React implicit for `fullWidth={true}`